libclamav/readdb.c

```diff
@@ -3116,16 +3134,12 @@ static int cli_loadcrt(FILE *fs, struct cl_engine *engine, struct cli_dbio *dbio
 
         ca.hashtype = CLI_HASHTYPE_ANY;
         crtmgr_add(&(engine->cmgr), &ca);
-        free(subject);
-        free(pubkey);
-        subject = pubkey = NULL;
+
+        FREE(pubkey);
     }
 
 end:
```

Comment:
It's a bit outside of the scope, but I would recommend changing this label to `done:` as well while cleaning up this function.